### PR TITLE
DOC-4690: Define basic install resources

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by davidcurrie — APPROVED** [review]
LGTM :shipit:

**Review by vadanx — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/deploy-corda-cluster.md [hunk 1]
```diff
@@ -124,30 +124,53 @@ Depending on your application workload, you may require additional replicas.
...
+      requests:
+        memory: 2048Mi
+      limits:
+        memory: 2048Mi
 ```
 As with the number of replicas, you may need to adjust these values based on testing with your actual application workload.
 
+#### Recommended Infrastructure
+
+When it comes to AWS topology, we recommend the following initial configuration:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/deploy-corda-cluster.md:162]:
```suggestion
Regarding AWS topology, we recommend the following initial configuration:
```

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/deploy-corda-cluster.md [hunk 2]
```diff
@@ -124,30 +124,53 @@ Depending on your application workload, you may require additional replicas.
...
+
+* Kubernetes: For a cluster with a single replica of each worker, a Kubernetes cluster with two `t3.2xlarge` nodes is
+a reasonable starting point. For a cluster with three replicas of each worker, extend that to four nodes.
+
+* RDS PostgreSQL: `db.r5.large` instance size is sufficient for both a Corda cluster with a single replica of each worker
+and three replicas of each worker, subject to the persistence requirements of any CorDapp running in the cluster.
+
+* MSK: For a cluster with a single replica of each worker, and a topic replica count of three, a Kafka cluster of three
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/deploy-corda-cluster.md:170]:
```suggestion
* MSK: For a cluster with a single replica of each worker and a topic replica count of three, a Kafka cluster of three
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**